generator/generator_tests/osm_type_test.cpp

```diff
@@ -410,10 +410,9 @@ UNIT_CLASS_TEST(TestWithClassificator, OsmType_Capital)
 
     auto const params = GetFeatureBuilderParams(tags);
 
-    TEST_EQUAL(params.m_types.size(), 3, (params));
+    TEST_EQUAL(params.m_types.size(), 2, (params));
     TEST(params.IsTypeExist(GetType({"place", "city", "capital", "2"})), (params));
     TEST(params.IsTypeExist(GetType({"boundary", "administrative", "4"})), (params));
-    TEST(params.IsTypeExist(GetType({"place", "city", "capital", "4"})), (params));
```

Comment:
its not clear why "place", "city", "capital", "2" matches
and "place", "city", "capital", "4" doesn't

Reply:
Because admin_level=2 is _better_ (more important) than 4.